client/store/redux/cart.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+//  ACTION TYPES
+const ADD_TO_CART ='ADD_TO_CART';
+const UPDATE_QTY = 'UPDATE_QTY';
+
+// ACTION CREATORS
+const addToCart = (cartProduct) => ({
+  type: ADD_TO_CART,
+  payload: cartProduct
+});
+
+const updateQty = (cartProduct, qty) => ({
+  type: UPDATE_QTY,
+  cartProduct,
+  qty
+});
+
+//  Thunks
+
+export const addItemToCart = (cartProduct) => async dispatch => {
+        const { data } = await axios.post(`/api/orders`, cartProduct)
```

Comment:
Before writing your thunk creator, you need a POST /api/orders route that works and returns the correct seed data in postman/insomnia